templates/login/login.html

```diff
@@ -0,0 +1,71 @@
+{% load static %}
+
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>AcadeMe</title>
+    <link rel="stylesheet" href="{%  static  'css/landingStyle.css'  %}" />
+    <link
+      href="//maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css"
+      rel="stylesheet"
+      id="bootstrap-css"
+    />
+    <script src="//maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js"></script>
+    <script src="//cdnjs.cloudflare.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
+  </head>
+
+  <body>
+    <link
+      href="//maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css"
+      rel="stylesheet"
+      id="bootstrap-css"
+    />
+    <script src="//maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js"></script>
+    <script src="//cdnjs.cloudflare.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
+    <!------ Include the above in your HEAD tag ---------->
+
+    <div class="wrapper fadeInDown">
+      <div id="formContent">
+        <!-- Tabs Titles -->
+
+        <!-- Icon -->
+        <div class="fadeIn first">
+          <div class="col-md-5">
+            <img
+              src="{%  static   'img/logo.jpg' %}"
+              style="width: 270%; height: 260%"
+              alt="home"
+            />
+          </div>
+        </div>
+
+        <!-- Login Form -->
+        <form>
+          <input
+            type="text"
+            id="login"
+            class="fadeIn second"
+            name="login"
+            placeholder="login"
+          />
+          <input
+            type="text"
+            id="password"
+            class="fadeIn third"
+            name="login"
+            placeholder="password"
+          />
+          <input type="submit" class="fadeIn fourth mt-2 mb-2" value="Log In" />
+        </form>
+
+        <!-- Remind Passowrd -->
+        <div id="formFooter">
+          <a class="underlineHover" href="#">Forgot Password?</a>
+          <a class="underlineHover" href="{% url 'register' %}"
+            >Not an AcadeMe user? Register HERE</a
```

Comment:
probably wanted to make HERE a link and not the whole sentence, didn't you?

Reply:
What do you mean?
the whole sentence is a link

Reply:
Register HERE.... the HERE should be a link, isnt it?

Reply:
The whole line is a link right now but I can make HERE only a link 
does it matter?